### PR TITLE
fix: replace deprecated dependencies

### LINKED ISSUE #11: I get this warning regarding random-prime package...
Warning: require() of ES modules is not supported.
require() of /opt/storage/appdir/node_modules/random-prime/index.js from /opt/storage/appdir/node_modules/encrypted-smiley-secure-protocol/index.js is an ES module file as it is a .js
 file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename /opt/storage/appdir/node_modules/random-prime/index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /opt/storage/appdir/no
de_modules/random-prime/package.json.

### PR DESCRIPTION
**Pull request by skokov3812**
remove packages: big-integer, int64-buffer, random-prime

### CODE REVIEW

**Comment by kybarg** [comment]
@skokov3812 seems like this package is going to be deprecated according to developers message https://www.npmjs.com/package/bigint-secrets
>bigint-secrets
IMPORTANT! This package has been superseded by bigint-crypto-utils. Please install that package instead.
Seems like this is alternative https://www.npmjs.com/package/bigint-crypto-utils

**Comment by skokov3812** [comment]
@kybarg Can you check that it works correctly on a physical device?

**Comment by kybarg** [comment]
@skokov3812 ok, sure

**Comment by kybarg** [comment]
blocked by https://github.com/juanelas/bigint-crypto-utils/issues/8

**Comment by kybarg** [comment]
@skokov3812 checked with fixed package but it hangs on **SET_GENERATOR **
```
COM <- 7f0001116608 SYNC 0
COM -> 7f0001f0200a | RAW
COM -> 7f0001f0200a SYNC
{ success: true, status: 'OK', command: 'SYNC', info: {} }
COM <- 7f000206061b94 HOST_PROTOCOL_VERSION 0
COM -> 7f0001f0200a | RAW
COM -> 7f0001f0200a HOST_PROTOCOL_VERSION
{ success: true,
  status: 'OK',
  command: 'HOST_PROTOCOL_VERSION',
  info: {} }
COM <- 7f80094a00a93c SET_GENERATOR 0
TIMEOUT SET_GENERATOR
COM <- 7f00094a0096bc SET_GENERATOR 0
```

**Comment by kybarg** [comment]
@skokov3812 
```
COM <- 7f0001116608 SYNC 0
COM -> 7f0001f0200a | RAW
COM -> 7f0001f0200a SYNC
{ success: true, status: 'OK', command: 'SYNC', info: {} }
COM <- 7f000206061b94 HOST_PROTOCOL_VERSION 0
COM -> 7f0001f0200a | RAW
COM -> 7f0001f0200a HOST_PROTOCOL_VERSION
{ success: true,
  status: 'OK',
  command: 'HOST_PROTOCOL_VERSION',
  info: {} }
COM <- 7f80094ab3fe0000000000001045 SET_GENERATOR 0
COM -> 7f8001f02380 | RAW
COM -> 7f8001f02380 SET_GENERATOR
{ success: true,
  status: 'OK',
  command: 'SET_GENERATOR',
  info: {} }
COM <- 7f00094b45e0000000000000ba9e SET_MODULUS 0
COM -> 7f0001f0200a | RAW
COM -> 7f0001f0200a SET_MODULUS
{ success: true, status: 'OK', command: 'SET_MODULUS', info: {} }
COM <- 7f80094c82820000000000003fbb REQUEST_KEY_EXCHANGE 0
COM -> 7f8009f0e512000000000000f063 | RAW
COM -> 7f8009f0e512000000000000f063 REQUEST_KEY_EXCHANGE
{ success: true,
  status: 'OK',
  command: 'REQUEST_KEY_EXCHANGE',
  info: { key: [ 229, 18, 0, 0, 0, 0, 0, 0 ] } }
(node:4208) UnhandledPromiseRejectionWarning: TypeError: data.readBigInt64LE is not a function
    at SSP.createHostEncryptionKeys (E:\Projects\encrypted-smiley-secure-protocol\src\index.js:262:36)
    at SSP.parsePacket (E:\Projects\encrypted-smiley-secure-protocol\src\index.js:251:14)
    at EventEmitter.eventEmitter.once.buffer (E:\Projects\encrypted-smiley-secure-protocol\src\index.js:198:24)
    at Object.onceWrapper (events.js:286:20)
    at EventEmitter.emit (events.js:198:13)
    at SerialPort.port.on.buffer (E:\Projects\encrypted-smiley-secure-protocol\src\index.js:98:26)
    at SerialPort.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at SerialPort.Readable.push (_stream_readable.js:224:10)
(node:4208) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4208) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

File: src/index.js [hunk 1]
```diff
@@ -254,9 +259,9 @@ module.exports = class SSP extends events {
   }
   createHostEncryptionKeys(data) {
     if (this.keys.key === null) {
-      this.keys.slaveIntKey = bigInt(Int64LE(data).toString());
-      this.keys.key = this.keys.slaveIntKey.pow(this.keys.hostRandom).mod(this.keys.modulusKey);
-      this.encryptKey = Int64LE(this.keys.fixedKey, 16).buffer.concat(Int64LE(this.keys.key).buffer);
+      this.keys.slaveIntKey = Buffer.from(data).readBigInt64LE();
+      this.keys.key = this.keys.slaveIntKey ** this.keys.hostRandom % this.keys.modulusKey;
+      this.encryptKey = int64LE(this.keys.fixedKey).concat(int64LE(this.keys.key));
```
Comment (kybarg) [inline src/index.js:264]:
```
this.encryptKey = Buffer.concat([int64LE(this.keys.fixedKey), int64LE(this.keys.key)]);
```

Comment (kybarg) [reply src/index.js:264]: but then seems like ther is an issue with encryption

**Comment by kybarg** [comment]
@skokov3812 thank you

**Pull request merged**